Procfile

```diff
@@ -1 +1 @@
-worker: node app.js
+worker: node --max-old-space-size=460 app.js
```

Comment:
This doesn't look like it's actually required for the fix, but it's a [recommended best practice](https://devcenter.heroku.com/articles/node-best-practices#avoid-garbage) for the 512mb memory containers